stage1/init/common/kvm_mount.go

```diff
@@ -55,6 +56,12 @@ const (
 	stage1MntDir = "/mnt/"
 )
 
+var log *rktlog.Logger
+
+func init() {
+	log = rktlog.New(os.Stderr, "kvm mount", false)
```

Comment:
I feel uneasy seeing a "kvm mount" prefix in a common package. Also, I don't see if we call `SetDebug` on the logger. Does that mean that we are never going to see full error stack?